components/ResultListPage/PlayAgainButton/index.js

```diff
@@ -0,0 +1,50 @@
+import styled from "styled-components";
+import { useRouter } from "next/router";
+import { useState } from "react";
+
+export default function PlayAgainButton({
+  currentRound,
+  setCurrentRound,
+  setRoundPoints,
```

Comment:
We don't pass the setter functions directly, but wrap them into a `handleCurrentRound` functions which can be passed as `onHandleCurrentRound` to the `PlayAgainButton`.